packages/stacks-docs/product/components/inputs.html

```diff
@@ -262,7 +264,7 @@ <h1 class="fs-headline1 fw-normal mb16">
                     <label class="flex--item s-label" for="example-success">Username</label>
                     <div class="d-flex ps-relative">
                         <input class="s-input" id="example-success" type="text" value="aaronshekey" aria-describedby="example-success-desc" />
-                        {% icon "Checkmark", "s-input-icon" %}
+                        {% icon "Check", "s-input-icon" %}
```

Comment:
I could've sworn I couldn't find this on https://beta.stackoverflow.design/product/foundation/icons/ this week. Thanks for fixing